README.md

```diff
@@ -1,26 +1,70 @@
-# apollo-link-rest
+---
+title: REST Link
+---
 
-## Problem to solve
-You have an existing REST set of services that you have worked on for a long time. However the limitations of it are piling up and you think GraphQL is a great solution to make your application faster and easier to develop. With `apollo-link-rest`, you can start trying out GraphQL without a full server. You can use it to prototype and even use it for third party services that don't yet have a GraphQL endpoint.
+## ⚠️ This library is under active development ⚠️
 
-Apollo Link Rest lets you query traditional REST API endpoints while writing GraphQL. It is designed to work with Apollo Client, but can even be used on its own with Apollo Link.
+This library is under active development. For information on progress check out
```

Comment:
Looking at the other READMEs, none of them have hard wrapping. This isn't a blocker, but would you mind if we turned off hard-wrapping on your formatting here?

Reply:
Good call! 👌